x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_overview_logic.ts

```diff
@@ -7,43 +7,58 @@
 
 import { kea, MakeLogicType } from 'kea';
 
-import { flashAPIErrors } from '../../../shared/flash_messages';
+import { i18n } from '@kbn/i18n';
+
+import { flashAPIErrors, setSuccessMessage } from '../../../shared/flash_messages';
 
 import { HttpLogic } from '../../../shared/http';
 import { EngineLogic } from '../engine';
 
 import { CrawlerData, CrawlerDataFromServer, CrawlerDomain } from './types';
 import { crawlerDataServerToClient } from './utils';
 
+export const DELETE_DOMAIN_MESSAGE = (domainUrl: string) =>
```

Comment:
I'd consider starting a `constants.ts` or `i18n.ts` - when I open a logic file I'm generally not a huge fan of immediately having to scroll/skip past a bunch of constants/strings to get to the actual logic

Reply:
let me think on this, I prefer to have constants in the same file if they're not being shared tbh